src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const totalPopulation = document.querySelector('span.total-population');
+const averagePopulation = document.querySelector('span.average-population');
+const populationByCountries = document.querySelectorAll('span.population');
+
+function removeCommas(string) {
+  return string.replaceAll(',', '');
+}
+
+function getTotalPopulation() {
+  let total = 0;
+
+  for (const countryPopulation of populationByCountries) {
+    const data = countryPopulation.textContent.trim();
+    const population = +parseInt(removeCommas(data));
+
+    total += population;
+  }
```

Comment:
In this case better to use reduce, because it`is typical case where we using this method, and you can practice it